### PR TITLE
Upgrade to java-common 0.12.7

### PR DESCRIPTION
**Pull request by frenchfrywpepper**
This version of java-common no longer supplies a version for the tracer so that is now implemented in this project.

### CODE REVIEW

File: lightstep-tracer-jre/src/main/java/com/lightstep/tracer/jre/Version.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package com.lightstep.tracer.jre;
+
+class Version {
+    static final String LIGHTSTEP_TRACER_VERSION = "0.12.5";
```
Comment (danielamiao) [inline lightstep-tracer-jre/src/main/java/com/lightstep/tracer/jre/Version.java]: 0.12.7 here?

Comment (danielamiao) [inline lightstep-tracer-jre/src/main/java/com/lightstep/tracer/jre/Version.java]: nvm this, I see if you run the `inc-version` script it upgrades it to 0.12.7

Comment (frenchfrywpepper) [inline lightstep-tracer-jre/src/main/java/com/lightstep/tracer/jre/Version.java]: It should match what is in the pom.xml, which is 0.12.5. Running inc-version should modify it to 0.12.6

Comment (danielamiao) [inline lightstep-tracer-jre/src/main/java/com/lightstep/tracer/jre/Version.java]: ah got it, I forget we talked about the versions moving independently of common now, thanks!

**Pull request merged**